salt_resources/states/basebox/miniconda.sls

```diff
@@ -0,0 +1,9 @@
+download_miniconda_installer:
+  cmd.run:
+    - name: wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O /home/vagrant/miniconda.sh
+    - unless: test -f /usr/local/bin/miniconda/bin/conda
+
+ensure_miniconda_is_installed:
+  cmd.run:
+    - name: bash /home/vagrant/miniconda.sh -b -p /usr/local/bin/miniconda
```

Comment:
Reading this it seemed fine at first, but I'm running into trouble using it. I think we should run this as `{{ grains['deescalated_user'] }}` and possibly remove the manual -p flag to put the miniconda[2,3] folder in the normal place.

ATM, trying this out I get:
```
webserver@debian:~$ source activate miniconda_env
bash: activate: No such file or directory
```
and basically the same thing on the vagrant user. I think miniconda isn't great at using the custom install location. I'm hoping this change would make activating the env work again.

Apart from the fix, the ~/miniconda[2,3] is the traditional spot, so we should probably use it, lest we confuse someone who's looking for it.